pkg/api/customization/app/app.go

```diff
@@ -151,7 +151,8 @@ func (w Wrapper) ActionHandler(actionName string, action *types.Action, apiConte
 		if valuesYaml != nil {
 			obj.Spec.ValuesYaml = convert.ToString(valuesYaml)
 		}
-
+		// indicate this a user driven action
+		obj.Annotations["userAction"] = "true"
```

Comment:
can you change the name of the annotation to `app.project.cattle.io/userTriggeredAction`